spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -134,10 +173,14 @@ function onChanOpenInit(
   // assert that version is "ics20-1" or empty
   // if empty, we return the default transfer version to core IBC
   // as the version for this channel
-  abortTransactionUnless(version === "ics20-1" || version === "")
+  abortTransactionUnless(version === "ics20-2" || version === "ics20-1" || version === "")
```

Comment:
Line 160: "- The version string is `ics20-1`" 
Should be changed accordingly to line 176: 
"-The version string is `ics20-1` or `ics20-2`